Include/cpython/fileutils.h

```diff
@@ -32,6 +32,14 @@ PyAPI_FUNC(int) _Py_EncodeLocaleEx(
     int current_locale,
     _Py_error_handler errors);
 
+#ifdef HAVE_NON_UNICODE_WCHAR_T_REPRESENTATION
+PyAPI_FUNC(int) _Py_IsLocaleUnicodeWchar(void);
+
+PyAPI_FUNC(char32_t*) _Py_convert_wchar_t_to_UCS4(
+    const wchar_t* u,
+    Py_ssize_t size);
+#endif
```

Comment:
I don't think that these functions should be used outside Python. Please move them to Include/internal/pycore_fileutils.h and replace PyAPI_FUNC() with extern.

Reply:
Thanks, I moved them there as you suggested.